### PR TITLE
add a flag to control whether inherit owner annotations when podgroup…

### LINKED ISSUE #2460: performance degration when scheduling a large number of native k8s workload
**What happened**:
Use volcano as a scheduler to schedule a large number of deploys, the performance is not as good as `kube-scheduler`.

**What you expected to happen**:
Performance is as good as `kube-scheduler`

**How to reproduce it (as minimally and precisely as possible)**:
test scheduling time using 200 deployments each with 10 pods inside

**Anything else we need to know?**:

**Environment**:
- Volcano Version: 1.6.0
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by elinx**
Fix #2460

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
Do you link a wrong issue? What's the use case to inherit owner annotations?

**Comment by elinx** [comment]
> Do you link a wrong issue? What's the use case to inherit owner annotations?

It's the right issue. This feather has already been in volcano, I just add a flag so that people can disable it.

File: cmd/controller-manager/app/options/options.go [hunk 1]
```diff
@@ -82,6 +84,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.EnableHealthz, "enable-healthz", false, "Enable the health check; it is false by default")
 	fs.DurationVar(&s.DetectionPeriodOfDependsOntask, "detection-period-of-dependson-task", defaultDetectionPeriodOfDependsOntask, "It indicates how often to detect the status of dependent tasks."+
 		"e.g. --detection-period-of-dependson-task=1s")
+	fs.BoolVar(&s.InheritOwnerAnnotations, "inerit-owner-annotations", true, "Enable inherit owner annotations for pods when create podgroup; it is enabled by default")
```
Comment (william-wang) [inline cmd/controller-manager/app/options/options.go:87]: s/inerit-owner-annotations/inherit-owner-annotations/g

Comment (elinx) [reply cmd/controller-manager/app/options/options.go]: fixed

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**